cluster.go

```diff
@@ -906,6 +916,12 @@ func (c *Cluster) Recover(h *cid.Cid) (api.GlobalPinInfo, error) {
 	return c.globalPinInfoCid("TrackerRecover", h)
 }
 
+// RecoverLocal triggers a recover operation for a given Cid in this peer only.
+// It returns the updated PinInfo, after recovery.
+func (c *Cluster) RecoverLocal(h *cid.Cid) (api.PinInfo, error) {
```

Comment:
This is a general point of curiosity that applies to a few of the tracker functions: what's the value in adding this method to the cluster component, when the rpc_api server could just call `c.tracker.Recover(h)` directly?

Reply:
The point is maintaining parallelism between the Rest API and the Go API as to what is the user-facing interface that cluster offers. The rest API or the RPC implementation could call the tracker RPC methods directly, but as a user bringing up cluster programatically you would not be able to access the tracker to do the same (at least via the cluster object). It's easy to do, and avoids people relying on somewhat more internal interfaces.